tests/dispenser/TestFundCommand.test_fund_command_invalid_args.approved.txt

```diff
@@ -0,0 +1,4 @@
+Usage: algokit dispenser fund [OPTIONS]
+Try 'algokit dispenser fund -h' for help.
+
+Error: Missing option '--receiver' / '-r'.
```

Comment:
I don't think we need a test that checks invalid args, that's testing click itself at that point?

Reply:
Its also a safety guard against potential misconfiguration in changing required flag on the option. We also do similar tests on commands like dispense. Feel free to unresolve if you want me to get rid of this one otherwise i still think there is value in this test